themes/vocabulary_theme/templates/macros/posts.html

```diff
@@ -21,10 +23,11 @@ <h4 class="b-header"><a class="blog-title" href="{{ post|url }}">{{ post.title }
 {% endmacro %}
 
 {% macro render_posts(posts) %}
-  <div class="columns is-multiline blog-entries">
+  <div class="columns is-multiline blog-entries" role="list">
```

Comment:
I feel like if we need a list, we should use an actual `ul` or `ol` and not try to bend a `div` into a list via a `role`